rust/tests/namespace.ts

```diff
@@ -759,16 +766,25 @@ describe("namespace", () => {
     console.log("cacheArtifactTxSig: %s", cacheArtifactResult.txid);
 
     const [namespace, _namespaceBump] = await pdas.getNamespacePDA(nsMint);
-    var nsData = await namespaceProgram.fetchNamespace(namespace);
+    let nsData = await namespaceProgram.fetchNamespace(namespace);
```

Comment:
https://github.com/raindrops-protocol/raindrops/pull/28/files#r945335531

Reply:
I think this is fine how it is personally

Reply:
changed